src/routes/schema/electionSchema.js

```diff
@@ -0,0 +1,5 @@
+import Joi from 'joi';
+
+export const GET_ELECTION_BY_ID_SCHEME = Joi.object().keys({
+    max: Joi.string().max(36),
```

Comment:
do you really want to max param here

Reply:
not really. just has adhered to the coding pattern.